README.md

```diff
@@ -63,7 +63,7 @@ $ unzip rocketmq-all-5.1.3-bin-release.zip
 
 Prepare a terminal and change to the extracted `bin` directory:
 ```shell
-$ cd rocketmq-all-5.1.3/bin
+$  cd rocketmq-all-5.1.3-bin-release/bin
```

Comment:
It would be better to remove the extra spaces at the beginning.

Reply:
@RongtongJin Done in the last commit